src/Avalonia.Controls/TextBox.cs

```diff
@@ -407,14 +407,18 @@ protected override void OnGotFocus(GotFocusEventArgs e)
             _presenter?.ShowCaret();
         }
 
+        private void ClearSelection()
```

Comment:
Can we actually make this public? I recently had need for something like this

Reply:
good idea... added comments and made public.

Reply:
Is setting SelectedText=null doing the same actually?